### PR TITLE
Updated Readme file

### PR DESCRIPTION
**Pull request by ShekharDhangar**
Updated Readme file

### CODE REVIEW

File: product-detail-page/product-page.html [hunk 1]
```diff
@@ -71,8 +71,8 @@
                         alt="">
```
Comment (Developers-codz) [inline product-detail-page/product-page.html]: provide value to alt

**Review by Developers-codz — APPROVED** [review]
looks good to merge

File: README.md [hunk 2]
```diff
@@ -1,2 +1,15 @@
 # shed-store
- A one stop for all your purchase
+
+### About
+SHED-STORE is  one stop platform for all your fashion purchase.
+
+### Preview
```
Comment (MayankKumar10) [inline README.md:6]: be little more description about your Site add pages link or images

Look into - https://readme.so/ 
for reference

**Review by MayankKumar10 — APPROVED** [review]
Look into these Everything else looks good  to me

File: landing-page/landing-page.css [hunk 3]
```diff
@@ -90,3 +89,13 @@
   height: 3rem;
   margin: 0.5rem 1rem;
```
Comment (b0llu) [inline landing-page/landing-page.css]: you can use px here if the value is less than 1 rem

File: product-list/product-list.css [hunk 4]
```diff
@@ -66,17 +65,18 @@
...
-    margin: 0 .25rem;
+    margin: 0 2.5px;
 }
 /* RESET BTN */
-.reset-btn{
-    font-size: 1.25rem;
-    padding: 0 .5rem;
+.clear-btn{
+    font-size: 1.5rem;
+    padding: 0 .75rem;
```
Comment (b0llu) [inline product-list/product-list.css]: use px here

Comment (ShekharDhangar) [reply product-list/product-list.css]: done

**Pull request merged**